app/src/main-process/menu/build-default-menu.ts

```diff
@@ -286,6 +286,13 @@ export function buildDefaultMenu(sharedProcess: SharedProcess): Electron.Menu {
     },
   }
 
+  const showUserGuide: Electron.MenuItemOptions = {
+    label: 'Show User Guide',
```

Comment:
The link mentions "GitHub Desktop User Guide**s**" - I think the text here should also be pluralized.